### DISCUSSION TITLE
[v2] Livewire infinite loop on component post updates in error pages

### DISCUSSION THREAD

**Discussion by wizzymore** [livewire/livewire#6404] [7. Bugs (archived)] [answered]
### Livewire version

v2.12.6

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://github.com/wizzymore/infinite-loop-example

### Steps To Reproduce

1. Clone the repo i provided as example
2. I added StartSession in the global middleware so we can have the csrf-token required for livewire
3. Go to a page that doesn't exist: `/404` as an example
4. See the infinite loop

### Additional information

Is livewire not supposed to be used in error pages?
This sucks cause i use the global layout in 404 pages and i have my navigation bar which contains the notifications of the logged in user and this functionality can't be used currently.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by wizzymore** [comment]
Bump

**Reply by wizzymore** [reply]
Bump again

**Comment by wizzymore** [comment]
A nice thing to point is that this happens only when the 404 page does not exist as a route.

But, if you have a route that has a parameter and you manually abort from the controller ( id wrong or no permissions etc ) the livewire component works.

**Comment by wizzymore** [comment, marked as answer]
So i figured it out, you need a fallback route for 404 pages. As i had a custom handling of errors i did not need this before.

To do this just publish the default laravel errors and do the following:
```php
Route::fallback(function () {
    return view('errors.404');
});
```

**Reply by wizzymore** [reply]
Well, this kind of works.

If you have a route that matches and doesn't find the model or you do an abort(404) it still fails and does the infinite loop, but meh, it's the best i got.

Idk why you guys made livewire bound to the original route's status, it makes no sense.

**Reply by riccardo993** [reply]
I have the same problem with a custom route binding. When I throw a ModelNotFound exception the loop starts. I'm on Livewire V3 and the problem persists. Having the fallback url only helps me to keep the user logged in.

**Reply by wizzymore** [reply]
I found a solution for that too, you can remove the model route bindings middleware from Livewire or simply not bind models to routes ( in the controller function definition )
